### PR TITLE
Amy cwc-4978 sdk dnclist example

### PR DESCRIPTION
**Pull request by amym-genesys**
Updated to follow standards and new blueprint format.
Switched trigger and class sections because couldn't save trigger until class existed. 
Tried to make steps generic to work for Classic and Lightning.

### CODE REVIEW

File: src/SDK/sdk-dnclist-example/README.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+- [Create an Apex trigger and class](#create-an-apex-trigger-and-class)
+  - [Create the DoNotCallManager class in Salesforce](#create-the-donotcallmanager-class-in-salesforce)
+  - [Create the Contact trigger in Salesforce](#create-the-contact-trigger-in-salesforce)
+- [Test your work](#test-your-work)
+
+## Create an internal DNC list in Genesys Cloud
+
+1. In your Genesys Cloud organization, create an internal DNC list.
+
+    The **assests/data/** folder contains an example .csv file that you use. For more information, see [Create a new internal DNC list](https://help.mypurecloud.com/?p=4107 "Opens the Create a new internal DNC list article").
```
Comment (nateb-genesys) [inline src/SDK/sdk-dnclist-example/README.md:54]: @amym-genesys assests > assets

File: src/SDK/sdk-dnclist-example/README.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+2. Copy and save the ID of the DNC list.
```
Comment (nateb-genesys) [inline src/SDK/sdk-dnclist-example/README.md:56]: @amym-genesys in the preview of the file, this line starting with 2. is being pulled up right after the "Create a new internal DNC list" link.

File: src/SDK/sdk-dnclist-example/README.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+1. Create a custom setting with following values:
+    >**Note**: If **Setting Type** is grayed out, enable **Manage List Custom Settings Type**. For more information, see [List Custom Setting is greyed out](https://help.salesforce.com/articleView?id=000317370&language=en_US&type=1&mode=1 "Opens List Custom Setting is greyed out in the Salesforce documentation") in the Salesforce documentation.
+
+    * **Object Name**: PureCloud_DNC_List
+    * **Setting Type**: List
+
+    For more information, see [Create Custom Settings](https://help.salesforce.com/articleView?id=cs_about.htm&type=5 "Opens Create Custom Settings in the Salesforce documentation") in the Salesforce documentation.
+
+2. Add a custom field to the custom setting with the following values:
```
Comment (nateb-genesys) [inline src/SDK/sdk-dnclist-example/README.md:74]: @amym-genesys the line starting with 2. is being pulled up to directly after "Salesforce documentation"

File: src/SDK/sdk-dnclist-example/README.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+    >**Note**: **Length** must be 36 or greater.
+
+    * **Data Type**: Text
+    * **Length**: 36
+    * **Field Name**: DNC_List_Id
+
+    For more information, see [Add Custom Settings Fields](https://help.salesforce.com/articleView?id=cs_add_fields.htm&type=5 "Opens Add Custom Settings Fields in the Salesforce documentation") in the Salesforce documentation.
+
+3. Add the ID of the DNC list to the custom field.
```
Comment (nateb-genesys) [inline src/SDK/sdk-dnclist-example/README.md:83]: @amym-genesys the line starting with 2. is being pulled up to directly after "Salesforce documentation"

File: src/SDK/sdk-dnclist-example/README.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+
+    g. Click **Save**.
+
+## Configure the Do Not Call field in Salesforce
+
+This field is hidden by default. You must make the field visible to profiles and then add the field to **Contact Layout**.
+
+1. Set **Field-Level Security** for the **Do Not Call** field to **Visible**.
+
+    a. Find the Contact field called **Do Not Call**.
```
Comment (nateb-genesys) [inline src/SDK/sdk-dnclist-example/README.md:105]: @amym-genesys bullets are not properly nesting

File: src/SDK/sdk-dnclist-example/README.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+
+    c. Under **Visible**, select the profiles that you want to be able to see the **Do Not Call** field.
+
+    d. Click **Save**.
+
+    For more information, see [Set Field-Level Security for a Single Field on All Profiles](https://help.salesforce.com/articleView?id=users_fields_fls.htm&type=5 "Opens Set Field-Level Security for a Single Field on All Profiles in the Salesforce documentation") in the Salesforce documentation.
+
+2. Add **Do Not Call** to **Contact Layout**.
+
+    a. Open the page layout for Contacts.
```
Comment (nateb-genesys) [inline src/SDK/sdk-dnclist-example/README.md:117]: @amym-genesys bullets are not properly nesting

**Review by nateb-genesys — CHANGES_REQUESTED** [review]
@amym-genesys this looks great and much more succinct. good catch on the order of the trigger and class. I created them in Visual Studio Code, so I don't think the order mattered since they were both pushed to the Salesforce instance at the same time. I called out a few formatting issues I'm seeing with bullets inline. I am seeing them in this preview - https://github.com/MyPureCloud/purecloud-for-salesforce-examples/blob/c4beb3209c969160bf30953bb205dc3e694d78a0/src/SDK/sdk-dnclist-example/README.md.
**edit** I take that back. I'm not seeing the indenting issues now. I'll mark those as resolved.

File: src/SDK/sdk-dnclist-example/README.md (hunk 1 above)
Comment (nateb-genesys) [inline src/SDK/sdk-dnclist-example/README.md:54]: @amym-genesys assets has an extra s

**Review by travis-cawthorn — APPROVED** [review]

**Review by nateb-genesys — APPROVED** [review]

**Pull request merged**